useful-scripts/importer-tests/importer.py

```diff
@@ -0,0 +1,217 @@
+import os
+import time
+import json
+import requests
+import argparse
+import csv
+import urllib3
+import datetime
+import random 
+import sys
+from pathlib import Path
+import logging
+from dotenv import load_dotenv
+import zipfile
+import subprocess
+import glob 
+from os.path import basename
+
+
+""" 
+Instructions
+
+- place a zip file (containing a data and a meta file) in an archive file called 'archive.zip' in the root of ~/importer-tests
+- move the .env.example to .env ('mv .env.example .env')
+- get a jwt token from Admin EES & place in .env file
+- Command to run importer script: pipenv run python importer.py 
+- use the -c option if you want to import more than one subject (i.e. 'pipenv run python importer.py -c 5')
+
+""" 
+
+
+parser = argparse.ArgumentParser(prog="python authenticate.py", description="Used to load test importer")
+parser.add_argument("-c", "--count",
+                    dest="count",
+                    default=1,
+                    type=int,
+                    help="num of subjects to upload")
+parser.add_argument("-e", "--env",
+                    dest="env",
+                    default='dev',
+                    choices=['local', 'dev'],
+                    type=str,
+                    help="the environment to run tests against")
+args = parser.parse_args()
+
+
+def create_publication(API_URL):
+    run_identifier = datetime.datetime.utcnow().strftime('%Y%m%d-%H%M%S')
+    print(f'Starting load test with RUN_IDENTIFIER: {run_identifier}')
+    response = requests.request(
+        "POST",
+        url=f'{API_URL}/api/publications',
+        headers={
+            "Content-Type": "application/json",
+            "Authorization": f"Bearer {os.getenv('JWT_TOKEN')}",
+        },
+        json={
+            "title": f"importer-test_{run_identifier}",
+            "topicId": "3cd51f1d-6d9a-4c1e-3bfd-08d8b3c1d3dc", 
```

Comment:
Might be good to have this as a variable in .env so we can match it to a Topic in our local environments

Reply:
Good idea I've moved the topic ID & the API_URL variables to .env variables now.